bluetooth: fast_pair: Fix Personalized Name documentation (#9482)
nrfconnect/sdk-nrf: samples/bluetooth/peripheral_fast_pair/README.rst

Testing Personalized Name extension is described in `Fast Pair Certification Guidelines for Personalized Name`_.

.. note::

>    To mitigate Android Personalized Name write issues, whenever you change the Personalized Name on an Android phone, you have to disconnect the phone from the Fast Pair Provider, put the Provider in not discoverable advertising mode and wait for the phone to reconnect and write the new Personalized Name.

Reviewer: ```suggestion
   To mitigate Android Personalized Name write issues, whenever you change the Personalized Name on an Android phone, perform the following:
    
   * Disconnect the phone from the Fast Pair Provider.
   * Add the Fast Pair Provider in not discoverable advertising mode.
   * Wait for the phone to reconnect.
   * Write the new Personalized Name.
```
